viewer/src/main/java/nl/b3p/viewer/stripes/EditFeatureActionBean.java

```diff
@@ -193,7 +198,7 @@ public Resolution edit() throws JSONException {
                 json.put("success", Boolean.TRUE);
             } while(false);
         } catch(Exception e) {
-            log.error(String.format("Exception editing feature", e));
+            log.error(String.format("Exception editing feature", e),e);
```

Comment:
The string doesn't have any format specifiers to format, it should just be
```
            log.error("Exception editing feature", e);
```

Reply:
Fixed in commit